tripal/tripald3
# watermark visualizations

This PR will add watermark/overlay image to visualization:
![charts](https://user-images.githubusercontent.com/15472253/55990004-d36a7d80-5c63-11e9-91dc-03ec489ae61a.gif)

To test:
1. update variable `applyWatermark = false;` and set the value to true.
in file: js/tripalD3.watermark.js

2. change `watermark.png` image in css/ to desired file but ensure to use the same filename.

3. update list in dir variable in tripalD3Watermark() function located in js/tripalD3.watermark.js.
To add:
If Tripald3 is implemented in this page `mysite.com/coolcharts/awesomegene.php`, add the keyword `coolcharts` to the array dir.

4. save
5. clear cache
6. reload page
